assets/queries/k8s/allow_privilege_escalation_psp/test/positive_expected_result.json

```diff
@@ -0,0 +1,13 @@
+[
+	{
+		"queryName": "Allow Privilege Escalation PSP",
+		"severity": "MEDIUM",
+		"line": 9
+	},
+
+	{
+		"queryName": "Allow Privilege Escalation PSP",
```

Comment:
Change the query's name to be different. Maybe something like "PSP Allows Privilege Escalation".